scripts/actions/mobskills/rotten_stench.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Rotten Stench
+-- Lowers physical and magic accuracy of enemies within a fan-shaped area originating from the caster.
```

Comment:
This is the description from https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit#gid=57955395 but it's not technically a cone so not sure if the fan-shaped area comment applies?

Reply:
Probably just update the wording to clarify. 

Btw you can rebase your branch to get the CI failure resolved

Reply:
Updated to reflect that it's an AoE not a cone. Also rebased.